uptime-service-backend/README.md

```diff
@@ -2,105 +2,153 @@
 
 A backend service for delegation program where participating nodes report stats about their activity.
 
-## TL;DR
+
+> **Note** Currently MF does not have chart repository. To install this chart i.e. with helmfile you need to reffer to it following ways:
 
 ```console
-git clone https://github.com/MinaFoundation/helm-charts
-cd helm-charts/uptime-backend-service
-helm install RELEASE_NAME ./ --namespace NAMESPACE
+# helmfile.yaml
+<..>
+releases:
+  - name: network_name
+    chart: git::ssh://git@github.com/MinaFoundation/helm-charts.git@uptime-service-backend?ref=main
+<..>
 ```
 
 ## Prerequisites
 
-- Kubernetes 1.12+
-- Helm 3.1.0
+Before installing this Helm chart, you should have the following prerequisites:
 
-## Installing the Chart
+ - Access to Kubernetes cluster
+ - Helm installed on your local machine
+ - Basic knowledge of Kubernetes and Helm
+ - Access to https://github.com/MinaFoundation/helm-charts
+ - Optional: helmfile to install this chart
 
-To install the chart with the release name `RELEASE_NAME`:
+## Installation
 
-```console
-helm install RELEASE_NAME ./ --namespace NAMESPACE
+To install this Helm chart, the easiest is to create a helmfile.yaml with needed values and run:
+```bash
+$ helmfile template
+$ helmfile apply
+ ```
+
+Or use helmfile only to generate resources and apply them with kubectl like so:
+
+```bash
+$ helmfile template | kubectl -f -
 ```
 
-The command deploys `uptime-service-backend` on the Kubernetes cluster in the default configuration. The [Parameters](#parameters) section lists the parameters that can be configured during installation.
+You can get some inspiration from helmfiles in `examples` folder.
 
-## Uninstalling the Chart
+Verify that the chart is deployed successfully:
 
-To uninstall/delete the `RELEASE_NAME` deployment:
+```bash
+helmfile status #although kubectl probably would give better insights.
+```
 
-```console
-helm delete RELEASE_NAME
+## Configuration
+
+To get all available values in cloned `helm-charts` do:
+
+```bash
+helm show values ./uptime-service-backend
 ```
 
-The command removes all the Kubernetes components associated with the chart and deletes the release.
+The following table lists the configurable parameters of the `uptime-service-backend` chart and its common default values.
 
-> **Note:** For `keyspace` storage backend helmchart sets default certificate located in `/database/cert/sf-class2-root.crt`. However this could be 
+### Container parameters
```

Comment:
**Nitpick:** I would stay with the name `common` or name it `deployment` or `pod` rather than renaming it `container`